exporters/prometheus/src/main/java/io/opentelemetry/exporter/prometheus/MetricAdapter.java

```diff
@@ -86,7 +86,7 @@ private static String cleanMetricName(String descriptorMetricName) {
       case SUMMARY:
         return Collector.Type.SUMMARY;
     }
-    return Collector.Type.UNTYPED;
+    return Collector.Type.UNKNOWN;
```

Comment:
Glad metrics is alpha :P Makes me think we can't release this package as stable until the underlying simpleclient is also 1.0